### PR TITLE
Add travis ci script.

### PR DESCRIPTION
**Pull request by rcedwards**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+language: objective-c
+xcode_project: CoreDataStack.xcodeproj
+xcode_scheme: CoreDataStack
+xcode_sdk: iphonesimulator8.3
```
Comment (lyricsboy) [inline .travis.yml]: Do we need to be specific about the version here? Can we just use `iphonesimulator`?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+osx_image: xcode7
+script:
+- xcodebuild test -scheme 'CoreDataStack' -sdk iphonesimulator ONLY_ACTIVE_ARCH=NO
```
Comment (lyricsboy) [inline .travis.yml]: Why not use `xctool`?

Comment (lyricsboy) [inline .travis.yml]: Maybe it doesn't support Xcode 7 yet?

File: .travis.yml (hunk 1 above)
Comment (rcedwards) [inline .travis.yml]: Yeah we can remove that version specifier.

File: .travis.yml (hunk 2 above)
Comment (rcedwards) [inline .travis.yml]: I'm not sure. Let me give it a shot.

Comment (rcedwards) [inline .travis.yml]: @lyricsboy Yeah looks like they are still working on 7.0 support.

https://github.com/facebook/xctool/issues/528

**Comment by rcedwards** [comment]
@lyricsboy merging for now. I've created an issue ( #23 ) for xctool.

**Pull request merged**